web/tests/test_chapters.py

```
@@ -0,0 +1,562 @@
+
+from django.contrib.auth.models import User
+from django.test import Client, TestCase
+from django.urls import reverse
+from django.utils import timezone
+
+from web.models import Chapter, ChapterApplication, ChapterEvent, ChapterEventAttendee, ChapterMembership, ChapterResource
+
+
+class ChapterModelTests(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username="testuser", email="test@example.com", password="testpassword123")
```

Ruff (0.8.2)

Possible hardcoded password assigned to argument: "password"

(S106)